common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/io/impl/AbstractGeoInstanceWriter.java

```diff
@@ -65,6 +67,16 @@ public String getCustomEPSGPrefix() {
 		return getParameter(PARAM_CRS_CODE_FORMAT).as(String.class);
 	}
 
+	@Override
+	public void setWindingOrder(EnumWindingOrderTypes windingOrderType) {
+		setParameter(PARAM_UNIFY_WINDING_ORDER, Value.of(windingOrderType));
```

Comment:
Does it make any difference? Because I am getting the Winding Order as "getParameter(PARAM_UNIFY_WINDING_ORDER)
				.as(EnumWindingOrderTypes.class);".